src/sfizz/Voice.cpp

```diff
@@ -58,6 +58,17 @@ void sfz::Voice::startVoice(Region* region, int delay, int number, uint8_t value
             break;
         }
         waveOscillator.setWavetable(wave);
+
+        float phase;
+        float phaseParam = region->oscillatorPhase;
+        if (phaseParam >= 0) {
+            phase = phaseParam * (1.0f / 360.0f);
+            phase -= static_cast<int>(phase);
+        } else {
+            std::uniform_real_distribution<float> phaseDist { 0.0001f, 0.9999f };
+            phase = phaseDist(Random::randomGenerator);
```

Comment:
So the expected behavior is that for negative phase (especially -1) the  phase is random?

Reply:
Yes, this was observed in the software and is confirmed in the details written in the book